<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start devs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop devs</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDevs">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>47</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Devices:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditOutput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCreate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Random devs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonScanDB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan db</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDestroy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear devs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddDev">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Device</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetSrc">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>101</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSrc">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Src</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDst">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>170</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetDst">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>210</y>
      <width>101</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Dst</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMap">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUnmap">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>330</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Unmap</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>in:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxIns">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>47</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>out:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxOuts">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>47</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
